test/CMakeLists.txt

```diff
@@ -4,86 +4,77 @@ add_subdirectory(wcmp)
 
 add_subdirectory(error_handling)
 
-macro(run_test case)
+macro(make_cse_test case)
   get_filename_component(test_name ${case} NAME_WE)
   set(test_comment "! ")
   add_test(NAME ${test_name}.Run
-    COMMAND $<TARGET_FILE:CSE> -x${test_comment} -b -t1 "${test_name}"
-    WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
+  COMMAND $<TARGET_FILE:CSE> -x${test_comment} -b -t1 ${ARGN} "${test_name}"
```

Comment:
Does `ARGN` come through as a list or a separated string? In other words: Will this work if there is more than one additional argument?

Reply:
It comes as a list. It works if more arguments are given.

Reply:
I looked into this a little closer. It works as advertised, but I also discovered that a list in this context is automatically separated into separate strings for each argument. If you were to use `"${ARGN}"` it would be printed as a list (i.e., a semicolon separated string).